taichi/transforms/auto_diff.cpp

```diff
@@ -1008,6 +1008,9 @@ class MakeAdjoint : public ADTransform {
 
   void visit(GlobalStoreStmt *stmt) override {
     // erase and replace with global load adjoint
+    if (stmt->dest->is<ExternalPtrStmt>()) {
+      TI_NOT_IMPLEMENTED;
```

Comment:
How about raising `not supported` error in `void visit(ExternalPtrStmt *stmt)` ?

Reply:
I see